trippl3a/modules/ui.js

```diff
@@ -0,0 +1,89 @@
+import { ELEMENT_SELECTORS, PRIORITIES, STATUSES } from "../main.js"
+import { addClassSelector, changeStatus, deleteTask } from "./logic.js"
+
+export const render = (tasks) => {
+    const lowTasksWrapper = getElement(ELEMENT_SELECTORS.WRAPPER_LOW)
+    const highTasksWrapper = getElement(ELEMENT_SELECTORS.WRAPPER_HIGH)
+    getElement(addClassSelector(ELEMENT_SELECTORS.TASKS_HIGH)).remove()
+    getElement(addClassSelector(ELEMENT_SELECTORS.TASKS_LOW)).remove()
+    const lowPriorityTasks = filterTasks(tasks, PRIORITIES.LOW)
+    const highPriorityTasks = filterTasks(tasks, PRIORITIES.HIGH)
+    const lowPriorityTasksElement = document.createElement('div')
+    const highPriorityTasksElement = document.createElement('div')
+    lowPriorityTasksElement.classList.add(ELEMENT_SELECTORS.TASKS_LOW)
+    highPriorityTasksElement.classList.add(ELEMENT_SELECTORS.TASKS_HIGH)
+    lowPriorityTasks.forEach(task => {
+        lowPriorityTasksElement.append(createTaskElement(task))
+    })
+    highPriorityTasks.forEach(task => {
+        highPriorityTasksElement.append(createTaskElement(task))
+    })
+    lowTasksWrapper.append(lowPriorityTasksElement)
+    highTasksWrapper.append(highPriorityTasksElement)
+}
```

Comment:
Можно упростить избавившись от лишних filter и  forEach, так же не удалять и пересоздавать контейнер по новой.
К примеру так:
```suggestion
export const render = (tasks) => {
    document.querySelectorAll('.task').forEach(task => task.remove());
             
    tasks.forEach( task => {
        const currentTasksElement = document.querySelector(`.${task.priority}-priority-tasks`);
        currentTasksElement.append(createTaskElement(task));        
    })  
}
```

Reply:
Спасибо!
так намного короче, интересный способ сокращения кода